caffe2/python/onnx/backend.py

```diff
@@ -205,7 +205,7 @@ class Caffe2Backend(Backend):
     # so this interface MAY make BC-breaking changes.  Specify an
     # opset_version if you don't want this to version.
     @classmethod
-    def run_node(cls, node, inputs, device='CPU', opset_version=_known_opset_version):
+    def run_node(cls, node, inputs, device='CPU', opset_version=_known_opset_version, outputs_info=None):
         super(Caffe2Backend, cls).run_node(node, inputs, device)
```

Comment:
(I know this is not introduced in this diff) we should pass both device and outputs_info back to the parent class here (as keyword arg).

Reply:
Good catch. Changed.